docker/mysql/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:jessie
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN groupadd -r mysql && useradd -r -g mysql mysql
+
+# add gosu for easy step-down from root
+ENV GOSU_VERSION 1.7
+RUN set -x \
```

Comment:
Docker caches image during the build step by step.
If it's not required it's better to extract all bash scripts to separate file and execute them with single `RUN`